unit/README.md

```diff
@@ -0,0 +1,14 @@
+To run the unit tests, call `./run-unit-tests`.
```

Comment:
+1 to unit testing!  It would be good to also document (or point to external documentation) how to create new unit tests, since I'm sure there are other parts of Links that would benefit from this.

Reply:
Yes the OUnit documentation is admittedly not the best. I will try to maybe come around to this, but luckily doing simple unit testing is fairly straightforward (starts with `unit/unitTests.ml`). It would be nice if there was a better way to structure unit tests in submodules as currently I have all unit testing files start with UnitTests* to ensure they don't conflict with anything else.